README.rst

```diff
@@ -4,14 +4,111 @@
-required.
+Usage
+=====
+After installing marabunta will be available as a console command. To run properly it requires a migration file, which defines what should be updated/executed, and of course stuff like database pw. Most of these also can also be provided by an envvar and will be discussed in the options section.
+
+It is very much recomended to configure it, so that marabunta is ran automatically if odoo is started.
+For instance adding it to your docker entrypoint.
+
+Features
+========
+
+* backup: Marabunta allows for a backup command to be executed before the migration.
+* addon upgrades: Marabunta is able to install or upgrade odoo addons.
+* addon uninstall: Addons which were previously installed can be removed with marabunta.
+* operations: Allows to execute commands before or after upgrading modules.
+* modes: Modes allow the user to execute commands only on a certain environment. i.e. creation of demodata on a dev system.
+
+Versioning systems
+------------------
+Currently Marabunta allows for two different Versioning systems:
+The classic Major.Minor.Bugfix and the Five digits long versions for OdooMajor.OdooMinor.Major.Minor.Bugfix.
+Although the first marabunta version should always be called setup. This is a standin for all the versions till the release, to make it easier to keep marabunta versions and tags in sync.
+
+
+Options
+=======
+
+=================  ========  ========================   ===============================================
+option             shortcut  envvar                     purpose
+=================  ========  ========================   ===============================================
+--migration-file   -f        MARABUNTA_MIGRATION_FILE   Definition file for the migration.
+--database         -d        MARABUNTA_DATABASE         Database we want to run the migration on.
+--db-user          -u        MARABUNTA_DB_USER          Database user.
+--db-password      -w        MARABUNTA_DB_PASSWORD      Database password.
+--db-port          -p        MARABUNTA_DB_PORT          Database port (defaults to 5432).
+--db-host          -H        MARABUNTA_DB_HOST          Database port (defaults to localhost).
+--mode                       MARABUNTA_MODE             Mode marabunta runs in for different envs.
+--allow-serie                MARABUNTA_ALLOW_SERIE      Allow multiple versions to be upgraded at once.
+--force-version              MARABUNTA_FORCE_VERSION    Force the upgrade to a version no matter what.
+--web-host                   MARABUNTA_WEB_HOST         Interface to bind (defaults to 0.0.0.0).
+--web-port                   MARABUNTA_WEB_PORT         Port for the Webhost (defaults to 8069).
+--web-custom-html            MARABUNTA_WEB_CUSTOM_HTML  Path to custom html page to serve.
+=================  ========  =========================  ===============================================
+
+YAML layout & Example
+=====================
+Here is an Example migration file::
```

Comment:
Any possibility to put this in a standalone file that we could reference from here and read from parser.py?